finetuner/tuner/augmentation.py

```diff
@@ -16,22 +17,61 @@ def vision_preprocessor(
     :param width: image width.
     :param default_channel_axis: The color channel of the input image, by default -1, the expected input is H, W, C.
     :param target_channel_axis: The color channel of the output image, by default 0, the expected output is C, H, W.
+    :param phrase: phrase of experiment, either `train` or `validation`. At `validation` phrase, will not apply
```

Comment:
do you mean `phase`?

Reply:
good catch, this is something i constantly spell incorrectly :)

Reply:
both phase and phrase are greek, so :P